tests/specs/unit/app.spec.js

```diff
@@ -1,11 +0,0 @@
-'use strict'; //eslint-disable-line strict
```

Comment:
This file was removed as it lived in the wrong place, it is meant to be in `tests/client/`, a file already exists there which is why it was not moved.